### PR TITLE
Some new Fixes added for Quests on Mobile UI

### PR DESCRIPTION
**Pull request by molecula451**
##### Description

We recently found that our style sheets in our Quests code were a tiny bit unsorted for our Mobile UI/UX.

**Quest Start on Mobile UI looked Akward:**
emcharacter

**Character Class Unsorted and unresponsive**
character

##### Refers/Fixes

Unable to click submit -metacartel quest mobile UI [5418](https://github.com/gitcoinco/web/issues/5418)

1. - Added Comments on new Style Rules for better Code Maintenance
2. - Added New Style Rule (@ media) for Better Mobile UI (500px)
3. - Character Class unsorted on stage2 on Mobile UI aligned accordingly to the right border
4. - Button Line-Height px unresponsive on iPhone (fixed)
5. - Quests Rules fixed for better alignment on the iPhone

**Character should look Now:** 

fixedcharacter

**Quest Rules Style for Mobile UI was extremely bigger. We made some fixes so it looks like**

quest rules

**User was claiming that he actually could not submit it's answers because button was hitting the other element**
button1

- [ ] User should be able to submit it's answer on any stage. 

There are still many elements that are not well sorted on the site and that they need to be fix. I will be providing updates on a constantly basis aswell as on style document.

Guys also. If am leaving something left behind that I am not being able to look please let me know and I will take care

##### Testing

- Brave Browser
- VSCode
- Web3
- Gitcoin

### CODE REVIEW

File: app/assets/v2/css/quests.css [hunk 1]
```diff
@@ -1400,3 +1400,73 @@ body.green .tl{
   }
 }
 
+/* Responsive for iPhone 6/7/8/Plus & X */
+
+@media (max-width: 500px) {
```
Comment (thelostone-mc) [inline app/assets/v2/css/quests.css]: Can we change this to @media (max-width: 575.98px) { ... } ? 
This is what bootstrap responsive breakpoints which is usually what we end up following 

https://getbootstrap.com/docs/4.3/layout/overview/

File: app/assets/v2/css/quests.css [hunk 2]
```diff
@@ -1400,3 +1400,73 @@ body.green .tl{
...
+    margin-top: -5px;
+   }
+
+  #desc {
+    height: 80px;
+    overflow-y: scroll;
+  }
+  body.stage_4 #desc {
+    margin-top: -2px;
+    }
```
Comment (thelostone-mc) [inline app/assets/v2/css/quests.css]: re-indent ?

File: app/assets/v2/css/quests.css [hunk 3]
```diff
@@ -1400,3 +1400,73 @@ body.green .tl{
...
+  top: 35%;
+  }
+
+  body.stage_4 #header {
+    margin-top: -5px;
+   }
+
+  #desc {
+    height: 80px;
+    overflow-y: scroll;
```
Comment (thelostone-mc) [inline app/assets/v2/css/quests.css]: scroll / auto ?

File: app/assets/v2/css/quests.css [hunk 4]
```diff
@@ -1400,3 +1400,73 @@ body.green .tl{
...
+  #announce{
+    margin-top: 150px;
+  }
+  #announce img{
+    margin-bottom: 30px;
+  }
+  #leaderboard .inner_row {
+    height: 130px;
+  }
+}
```
Comment (thelostone-mc) [inline app/assets/v2/css/quests.css]: ^ could we ensure there is an 
- empty line after every style declaration 
- `xyz{` -> `xyz {`

File: app/assets/v2/css/quests.css (hunk 1 above)
Comment (molecula451) [reply app/assets/v2/css/quests.css]: Added.

File: app/assets/v2/css/quests.css (hunk 2 above)
Comment (molecula451) [reply app/assets/v2/css/quests.css]: Re-identation fully completed in the whole document.

**Comment by molecula451** [comment]
I will keep adding fixes and provide updates on this specific code so our quests are always fully responsive and intuitive.

If I am leaving something left behind. Please <let me know in advance

File: app/assets/v2/css/quests.css [hunk 5]
```diff
@@ -1555,7 +1555,7 @@
             }
             body.stage_4 #header {
                 margin-top: -2%;
-                font-size: 12px;
+                font-size: 8px;
```
Comment (molecula451) [inline app/assets/v2/css/quests.css]: Font-Size reduction so our user it's able to actually read the Question in it's phone

**Comment by molecula451** [comment]
@thelostone-mc expecting some feedback awesome mate

**Comment by thelostone-mc** [comment]
@molecula451 I udpated the indentation and stuff cause I wanted to get this in before this weeks deploy ^_^

**Comment by molecula451** [comment]
> @molecula451 I udpated the indentation and stuff cause I wanted to get this in before this weeks deploy ^_^

I was eager to know if you wanted to keep the 4-5 spaces latest indentation which was not from the original document. But it's all ok. Good stuff great friend

**Pull request merged**